core/server/master/src/main/java/alluxio/master/file/meta/InodeTreePersistentState.java

```diff
@@ -173,15 +181,11 @@ public Set<Long> getReplicationLimitedFileIds() {
    * @return the root of the inode tree
    */
   public InodeDirectory getRoot() {
```

Comment:
`Inode.initializeRoot()` calls this `getRoot()` and this `getRoot()` will return null. I'm not sure if this `null` will just be cached for some time. @elega WDYT?

Reply:
yeah you are correct.

In the LazyInitializer.initialize() method, we can probably throw an exception if the root doesn't exist. And we can catch the exception in the getRoot() method and return null in that case.